packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -67,7 +67,9 @@ const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = {
         ...highContrastForeground,
     },
     select_menu: {
-        background: neutralFillStealthRest,
+        background: (designSystem: DesignSystem): string => {
+            return designSystem.backgroundColor;
+        },
```

Comment:
You should be able to just do `background: backgroundColor,`.

Reply:
Actually that doesn't work.  Looking at this with Nick it became apparent that the new background color got inherited by the select options but not the select menu BG.  The fix is to actually set the menu background in the style as well as the context to cover both cases.